resources/i18n/EnchantmentCracker_zh_tw.properties

```
@@ -0,0 +1,106 @@
+
+program.name = Enchantment Cracker(附魔破譯器)
+
+# Tab tooltips on top of the GUI
+tab.enchantmentCracker = 附魔破譯器
+tab.enchantmentCalculator = 附魔計算機
+tab.about = 關於
+
+# Enchantment Cracker tab translations
+enchCrack.xpSeed1 = XP種子碼 1
+enchCrack.xpSeed1.tooltip = 所需的第一個XP種子碼
+enchCrack.xpSeed2 = XP種子碼 2
+enchCrack.xpSeed2.tooltip = 所需的第二個XP種子碼
+enchCrack.calculate = 計算種子
+enchCrack.bookshelves.tooltip = 附魔台周圍有效書架數量
+enchCrack.xpCost1.tooltip = 第一個附魔所需的附魔等級
+enchCrack.xpCost2.tooltip = 第二個附魔所需的附魔等級
+enchCrack.xpCost3.tooltip = 第三個附魔所需的附魔等級
+enchCrack.check = 檢測
+enchCrack.check.tooltip = 使用此功能來找尋能用的XP種子碼
+enchCrack.reset = 重置
+enchCrack.reset.tooltip = 重置當前儲存數據來破解新的XP種子碼
+enchCrack.impossible = 無匹配種子碼
+enchCrack.result = XP種子碼: %08X
+enchCrack.remaining = 可能的種子數量: %s
+enchCrack.remaining.thousand = 可能的種子碼數: %s仟個
+enchCrack.remaining.million = 可能的種子碼數: %s佰萬個
+enchCrack.remaining.billion = 可能的種子碼數量: %s十憶個
+enchCrack.progress = 進度: %02.0f%%
+
+# Enchantment Calculator tab translations
+enchCalc.item.tooltip = 想要附魔的物品
+enchCalc.material.tooltip = 想要附魔的物品的材料
+enchCalc.maxBookshelves.tooltip = 使用中的有效書架數
+enchCalc.calculate = 開始計算
+enchCalc.calculate.tooltip = <html>按下來計算你<i>可能</i>需要丟出多少物品來獲得想要的附魔</html>
+enchCalc.playerSeedNotFound = 你需要先破譯玩家種子碼!\n先去第一個欄位裡看看吧
+enchCalc.impossible = 無匹配結果
+enchCalc.noDummy = 無虛設數值
```

Not an expert in traditional chinese, but I think this should be more along the lines of:
```suggestion
enchCalc.noDummy = 無额外個附魔
```